reference-architectures/cloud_deploy_flow/main.tf

```diff
@@ -0,0 +1,176 @@
+# Configure the Google Cloud provider
```

Comment:
Commit the [Terraform dependency lock file](https://developer.hashicorp.com/terraform/language/files/dependency-lock) for reproducible installs.